### PR TITLE
feat(issue summary): Redesign issue summary UI

### PR DESCRIPTION
**Pull request by roaga**
New UI in both current and streamlined UI

Screenshot 2024-09-24 at 2 51 34 PM
Screenshot 2024-09-24 at 2 51 39 PM
Screenshot 2024-09-24 at 2 52 19 PM
Screenshot 2024-09-24 at 2 52 23 PM

### CODE REVIEW

**Review by jennmueng — APPROVED** [review]
🚄 LGTM

File: static/app/components/group/groupSummary.tsx [hunk 1]
```diff
@@ -125,91 +92,133 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
...
+      )}
+    </Wrapper>
   );
 }
 
+const Body = styled('div')`
+  padding-left: ${space(3)};
+  padding-right: ${space(3)};
+  padding-top: ${space(1)};
+  padding-bottom: ${space(1.5)};
```
Comment (scttcper) [inline static/app/components/group/groupSummary.tsx:172]: combine into `padding: `

File: static/app/components/group/groupSummary.tsx [hunk 2]
```diff
@@ -125,91 +92,133 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
...
+`;
+
+const Headline = styled('p')`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-right: ${space(0.5)};
+  flex-shrink: 0;
+  margin-top: ${space(3)};
```
Comment (scttcper) [inline static/app/components/group/groupSummary.tsx:181]: margin here is really odd. why is it needed?

image

File: static/app/components/group/groupSummary.tsx [hunk 3]
```diff
@@ -125,91 +92,133 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
...
+          )}
+          {data && expanded && <Headline>{data.headline}</Headline>}
+        </CollapsedRow>
+        <IconContainerRight>
+          <IconChevron direction={expanded ? 'up' : 'down'} />
+        </IconContainerRight>
+      </StyledTitleRow>
+      {expanded && (
+        <Body>
+          <div>
```
Comment (scttcper) [inline static/app/components/group/groupSummary.tsx:121]: useless div

File: static/app/components/group/groupSummary.tsx [hunk 4]
```diff
@@ -125,91 +92,133 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
...
 }
 
+const Body = styled('div')`
+  padding-left: ${space(3)};
+  padding-right: ${space(3)};
+  padding-top: ${space(1)};
+  padding-bottom: ${space(1.5)};
+`;
+
+const Headline = styled('p')`
```
Comment (scttcper) [inline static/app/components/group/groupSummary.tsx:175]: don't use paragraph for a heading

File: static/app/components/group/groupSummary.tsx [hunk 5]
```diff
@@ -125,91 +92,133 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
...
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```
Comment (scttcper) [inline static/app/components/group/groupSummary.tsx:178]: overflow ellipsis doesn't look like its working as expected 
image

File: static/app/components/group/groupSummary.tsx [hunk 6]
```diff
@@ -125,91 +92,133 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
...
+  margin-top: ${space(3)};
+`;
+
+const SummaryPreview = styled('p')`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  flex-grow: 1;
+  color: ${p => p.theme.subText};
+  margin-top: ${space(3)};
```
Comment (scttcper) [inline static/app/components/group/groupSummary.tsx:190]: i would figure out why you need padding here

File: static/app/components/group/groupSummary.tsx [hunk 7]
```diff
@@ -125,91 +92,133 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
...
-  );
-  padding: ${space(1.5)} ${space(2)};
+  padding: ${space(0.5)};
 `;
 
 const StyledTitleRow = styled('div')`
   display: flex;
   align-items: center;
   justify-content: space-between;
+  height: ${space(2)};
```
Comment (scttcper) [inline static/app/components/group/groupSummary.tsx:202]: prefer min height to height
```suggestion
  min-height: ${space(2)};
```

File: static/app/views/issueDetails/streamline/eventDetails.tsx [hunk 8]
```diff
@@ -79,6 +81,9 @@ export function EventDetails({
 
   return (
     <EventDetailsContext.Provider value={{...eventDetails, dispatch}}>
+      <Feature features={['organizations:ai-summary']}>
+        <GroupSummary groupId={group.id} groupCategory={group.issueCategory} />
```
Comment (scttcper) [inline static/app/views/issueDetails/streamline/eventDetails.tsx:85]: cam we wrap this with a `<ErrorBoundary mini>`

File: static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx [hunk 9]
```diff
@@ -150,6 +152,11 @@ export function EventDetailsContent({
       )}
       {hasStreamlinedUI && <TraceDataSection event={event} />}
       <StyledDataSection>
+        {!hasStreamlinedUI && (
+          <Feature features={['organizations:ai-summary']}>
+            <GroupSummary groupId={group.id} groupCategory={group.issueCategory} />
```
Comment (scttcper) [inline static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx:157]: cam we wrap this with a `<ErrorBoundary mini>`

**Pull request merged**